Rakefile

```diff
@@ -622,11 +623,40 @@ task :get_last_commit_date_shields do
 end # task
 
 
+# this shouldn't update very often, run crontab once a month
+desc "process dependency badge"
+task :process_dependency_badge do
+
+  destdir = File.join('assets', 'shields', 'dependencies', 'devel')
+  FileUtils.rm_rf destdir
+  FileUtils.mkdir_p destdir
+
+  puts "GENERATING BADGES:  bioc, devel"
```

Comment:
Better to iterate?
```
repos = ["bioc", "experiment", "workflows"]
repos.each do |repo|
  puts "GENERATING BADGES " + repo + ", devel"
  dependencyBadge(repo, destdir, false)
end
```
(also for release)

Reply:
I'll adjust and also change the other badge that does this too (I think rank)